core/src/main/java/io/kestra/core/repositories/LocalFlowRepositoryLoader.java

```diff
@@ -65,12 +68,9 @@ public void load(URL basePath) throws IOException, URISyntaxException {
         }
     }
 
-
     public void load(File basePath) throws IOException {
-        this.load(basePath, false);
-    }
-
-    public void load(File basePath, Boolean update) throws IOException {
+        Map<String, Flow> flowByUidInRepository = flowRepository.findAllForAllTenants().stream()
```

Comment:
I changed the implementation to a bulk fetch and always allow update for tests (to prevent tests from throwing errors due to flow already existing) but I just saw it's used by StandaloneCommand. Idk how critical it is to fetch the whole instance's flows. I can add a boolean to avoid bulk fetch in StandaloneCommand but I'm not sure, wdyt ?